minicluster/src/main/java/tachyon/master/LocalTachyonCluster.java

```diff
@@ -257,8 +257,10 @@ private void startMaster(TachyonConf testConf) throws IOException {
     // TODO(andrew): Move logic to the integration-tests project so that we can use instanceof here
     // instead of comparing classnames.
     if (ufs.getClass().getSimpleName().equals("LocalMiniDFSCluster")) {
-      testConf.set(Constants.UNDERFS_ADDRESS, ufs.getUnderFilesystemAddress() + mTachyonHome);
-      MasterContext.reset(testConf);
+      String ufsAddress = ufs.getUnderFilesystemAddress() + mTachyonHome;
+      testConf.set(Constants.UNDERFS_ADDRESS, ufsAddress);
+      MasterContext.getConf().set(Constants.UNDERFS_ADDRESS, ufsAddress);
+      mMasterConf = MasterContext.getConf();
```

Comment:
Actually I don't think you need line 263, as `mMasterConf` should still be `MasterContext.getConf()`